lib/shopify-cli/commands/authenticate.rb

```diff
@@ -0,0 +1,30 @@
+require 'shopify_cli'
+
+module ShopifyCli
+  module Commands
+    class Authenticate < ShopifyCli::Command
+      def call(args, _name)
+        spin_group = CLI::UI::SpinGroup.new
+        spin_group.add("Requesting access token...") do |spinner|
+          case args.shift
+          when 'ident'
```

Comment:
Can we just call it identity? I feel like this is hard to read and its only a couple extra characters.

Reply:
okay should I put 'auth' back to 'authenticate' as well?